server/server.go

```diff
@@ -588,7 +588,7 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 	}
 
 	legacyConftestExecutor := policy.NewConfTestExecutorWorkflow(ctxLogger, binDir, &terraform.DefaultDownloader{})
-	conftestExecutor := policy.NewConfTestExecutor(clientCreator, userConfig.GithubOrg)
```

Comment:
doesn't the `conftestExecutor` need organization info for all the different policy sets to get team members info? For example, if we have security and orchestration policy sets with github teams for each policy sets, wouldn't we need both the organization names in the conftest executor?

Reply:
for our implementation, the organization name would be the same across all policy sets. It really is just the Github organization name (ex. the github organization this repo belongs to is `lyft`). I debated putting it as a root level repo configuration but settled with the policy sets config.